### DISCUSSION TITLE
Sidkick64 not starting Raspberry 3A+ and Zero

### DISCUSSION THREAD

**Discussion by KeenVox** [frntc/Sidekick64#102] [Q&A] [answered]
Hi there,

I can't get my Sidekick64 to run. I checked all solderpoints, I put the A1 and D0 zero jumper. I can see the red light on the raspy and the backlight of the tft screen but then nothing. No splashscreen, no LED activity., nothing on the hdmi out.

I set the correct tft in the c64 conifg. I tried with a Raspi 3A+ and Zero . I changed in the config from Zero to 3A+ but nothing is happening. Anything what I am missing? I'm using the 250469 C64 board.

Thanks for any suggestions..

**Comment by SensoryTrash** [comment]
[Sk64-2022pi.zip](https://github.com/user-attachments/files/15990455/Sk64-2022pi.zip)
here is the correct file updated

**Comment by nosyn00b** [comment, marked as answer]
Hi, in my experience this happens if the SD card is not read correctly: check if the SD is FAT32-formatted! When my Rpi B3+ with the SK64 shield is booting correctly, a green LED will also flash just after the red one lights up, and then it flashes another time during Rpi's boot, just before the LCD comes alive and shows the SK64 logo (and the SK64 starts to work as a special cartridge).

**Reply by KeenVox** [reply]
Formating to FAT32 seemed to work. Thank you!

**Comment by SensoryTrash** [comment]
wasnt aware it was not an option i  have updated the listing

On Tue, Jul 2, 2024 at 2:22 PM nosyn00b ***@***.***> wrote:

> Hi, in my experience this happens if the SD card is not read correctly:
> check if the SD is FAT32-formatted! When my Rpi B3+ with the SK64 shield is
> booting correctly, a green LED will also flash just after the red one
> lights up, and then it flashes another time during Rpi's boot, just before
> the LCD comes alive and shows the SK64 logo (and the SK64 starts to work as
> a special cartridge).
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/frntc/Sidekick64/discussions/102#discussioncomment-9940038>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AW2ESIJLIAS6FTOCRAGDQNLZKLVVBAVCNFSM6AAAAABISPWK3KVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSNBQGAZTQ>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Eugenio Steele
Owner Operator
Plan Net
23 Kelsey Cres, Barrie, ON L4N 0J5
705-737-7695



For easy contact free payments i am accepting E-transfers & Paypal
when sending e-transfers please use email ***@***.***

*Also if your happy with my service please post a review here
***@***.***,-80.2228846,17z/data=!4m5!3m4!1s0x882a7bd9de1013d7:0xc4d171d597d5a537!8m2!3d44.5025625!4d-80.2206906>*
*Also follow me on Facebook here <https://www.facebook.com/SensoryTrash/>*
*Also using Remote Support here <https://fastsupport.gotoassist.com/>*

https://plannetcss.com/ For more information on Services Etc.

Barrie
